filecopy/operations/services/notification/client.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2022 Indoc Research
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation, either version 3 of the License, or any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along with this program.
+# If not, see http://www.gnu.org/licenses/.
+
+import logging
+from typing import List
+from typing import Optional
+
+from operations.models import Node
+from operations.models import NodeList
+from operations.services.notification.models import InvolvementType
+from operations.services.notification.models import Location
+from operations.services.notification.models import NotificationType
+from operations.services.notification.models import PipelineAction
+from operations.services.notification.models import PipelineNotification
+from operations.services.notification.models import PipelineStatus
+from operations.services.notification.models import Target
+from requests import Session
+
+logger = logging.getLogger(__name__)
+
+
+class NotificationServiceClient:
+    """Client for sending notifications into Notification Service."""
+
+    def __init__(
+        self,
+        endpoint: str,
+        include_nodes: NodeList,
```

Comment:
Here is also something wrong with typing. As I understand here will be:
```suggestion
        include_nodes: dict[str, Node],
```